common/metrics/config.go

```diff
@@ -399,18 +399,14 @@ func MetricsHandlerFromConfig(logger log.Logger, c *Config) MetricsHandler {
 		default:
 			return NewTallyMetricsHandler(
 				c.ClientConfig,
-				newPrometheusScope(
-					logger,
-					convertPrometheusConfigToTally(c.Prometheus),
-					&c.ClientConfig,
-				),
+				NewScope(logger, c),
```

Comment:
we should just remove default case and FrameworkTally case as well.

Reply:
Done.